packages/react-button/src/components/Button/useButton.ts

```diff
@@ -14,14 +15,18 @@ export const useButton = (
   options: ComposePreparedOptions,
 ): ButtonState => {
   const buttonRef = useRef<HTMLButtonElement | null>(null);
-
   useImperativeHandle(props.componentRef, () => ({
     focus: () => buttonRef.current?.focus(),
   }));
   useFocusRects(ref as React.RefObject<HTMLElement>);
 
+  const buttonBehaviorProps = useButtonBehavior(props, ref);
```

Comment:
```suggestion
  const buttonBehaviorProps = useButtonBehavior(props, buttonRef);
```

I expect there `buttonRef` as it will always exist

Reply:
Changed!